### DISCUSSION TITLE
Error Deploymen PythonAnywhere

### DISCUSSION THREAD

**Discussion by arsyiloo** [teamalgoritma/community#863] [Q&A] [answered]
Saya sudah mengikuti langkah deployment untuk telegram telebot di pythonanywhere, namun belum berhasil. 
<img width="553" alt="image" src="https://github.com/teamalgoritma/community/assets/92915805/e3a49024-54ee-4efd-9a6f-38e201de68b6">

Apakah kesalahannya di konfigurasi WSGI?
https://www.pythonanywhere.com/user/arsyil/shares/7735ec036b2d45d6a4c6487b28a09466/

<img width="395" alt="image" src="https://github.com/teamalgoritma/community/assets/92915805/da20488c-53b6-47f5-ab06-cd6f4cc048c1">
Pada bagian ini apakah ada kesalahan?

**Comment by finesaaa** [comment]
Iya betul pak, kesalahannya pada bagian WSGI filenya seperti yg di screenshoot oleh Bapak, seharusnya **from main import app as application**  bukan main_flask_app_file ya pak.

Contohnya seperti yang terlampir di Notebook Guide, coba itu dulu pak, nanti setelah coba diganti WSGI filenya coba di save lalu reload ya pak dan konfirmasi kembali disini :) 
Terima kasih.
![image](https://github.com/teamalgoritma/community/assets/57583780/28998356-e08d-475b-801c-4e556d3d3458)

**Comment by finesaaa** [comment, marked as answer]
Sebagai tambahan pak @arsyiloo 
Jika Bapak mengalami kendala, yaitu saat username.pythonanywhere.com telah berjalan dengan tampilan tulisan "Bot is Running", namun ketika mencoba mengetikkan chat pada TeleBot yang Bapak buat, bot tersebut tidak merespon, padahal TOKEN dan public_domain pada app.py telah benar.

Silahkan tambahkan threaded=False pada bot_local.py bagian # SETUP: TELEGRAM BOT API TOKEN, sehingga menjadi seperti berikut: `bot = telebot.TeleBot(TOKEN, threaded=False)`

Kemudian, lakukan reload ulang untuk username.pythonanywhere.com dan coba kembali TeleBot Bapak.

Best regards,
Team Algoritma.
